migration-engine/connectors/migration-connector/src/steps.rs

```diff
@@ -291,6 +290,16 @@ pub struct DeleteTypeAlias {
     pub type_alias: String,
 }
 
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct CreateSource {
+    pub name: String,
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct DeleteSource {
+    pub name: String,
```

Comment:
In the other step types, we have the convention to use the type of item instead of `name`. So for example `CreateModel` has a `model` field, `DeleteEnum` has an `enum` field, etc. It's not a big problem but I think `source` would be more consistent.